NASA-AMMOS/aerie-ui
# Procedural Constraints UI

resolves #1478

```___REQUIRES_AERIE_PR___="1596"```

To test:
1. Create a constraint, but click the new "JAR" button
2. Select a JAR file (any .jar works for showcasing the UI functionality)
3. Finish creating the constraint
4. Open/create a plan
5. Open the "Constraints" panel
6. Click the "Manage Constraints" button
7. Select the newly created constraint and click the "Update" button
8. Verify that the constraint shows up in the Constraints panel
9. Right click the constraint name
10. Click "Duplicate Invocation"
11. Verify that another entry with the same constraint name shows up
12. Right click the new constraint name
13. Click "Delete Invocation"
14. Verify that the instance is deleted


TODO:
- [x] Add constraint invocation argument diffing in order to determine if the displayed status is still relevant
